ceph.spec.in

```diff
@@ -713,6 +713,7 @@ Requires:	librados2 = %{_epoch_prefix}%{version}-%{release}
 Requires:	librgw2 = %{_epoch_prefix}%{version}-%{release}
 %if 0%{?rhel} || 0%{?fedora}
 Requires:	mailcap
+Requires:	gawk
```

Comment:
probably we can move it out of the `%if 0%{?rhel} || 0%{?fedora}` block. as I think gawk is very likely available on other distros using rpm packaging system.
also, could use 
```
%if 0%{?weak_deps}
Recommends:	gawk
%endif
```

but I don't feel strong about this.

Reply:
I went the weak_deps route.